### PR TITLE
#166 Refactored hardcoded library versions

### LINKED ISSUE #166: Hard coded librairy versions in some build.gradle
The build.gradle.kts of the modules driver-jdbc and driver-mongo both contain hard coded librairiy version : 
- implementation("org.jetbrains.kotlinx:kotlinx-serialization-runtime:0.20.0")
- testImplementation("com.h2database:h2:1.4.200")

### PR DESCRIPTION
**Pull request by zplusp**
Resolves #166

### CODE REVIEW

File: buildSrc/src/main/kotlin/Versions.kt [hunk 1]
```diff
@@ -8,4 +8,8 @@ object Versions {
     const val mongo = "4.0.13"
     const val mongoDriver = "3.11.2"
     const val clikt = "2.3.0"
+    const val h2Database = "1.4.200"
+    const val kotlinxSerializationRuntime = "0.20.0"
+    const val jongo = "1.4.1"
+    const val postgresql = "42.1.4"
```
Comment (Lysoun) [inline buildSrc/src/main/kotlin/Versions.kt:16]: I would love the values to be gathered depending on what they are about. For instance, gathering dependencies to mongo (mongo, mongoDRiver, kotlinxSerializationRuntime, jongo) and sql (h2database, porstgresql).

**Review by Lysoun — CHANGES_REQUESTED** [review]
Thank you so much for contributing to the project. As soon as you take my comment in account, I will make sure to merge the PR and add the label needed for the PR to be counted for the hacktoberfest!
Hope you found the project interesting and will use it in your projects or even contribute another time!

File: buildSrc/src/main/kotlin/Versions.kt (hunk 1 above)
Comment (zplusp) [reply buildSrc/src/main/kotlin/Versions.kt:16]: I'm sorry I didn't understand you. Do you mean it should be like `Versions.sql.postgresql` etc. or just reordering inside the file.

Comment (Lysoun) [reply buildSrc/src/main/kotlin/Versions.kt:16]: I meant reordering inside the file, putting some spaces. for example:

```kotlin
    const val kotlinxSerializationRuntime = "0.20.0"
    const val jongo = "1.4.1"

    const val h2Database = "1.4.200"
    const val postgresql = "42.1.4"
```

**Review by Lysoun — APPROVED** [review]

**Comment by driccio** [comment]
Hi, @zplusp thanks a lot for your contribution. We are pleased to see external contributors.
Feel free to suggest some improvements. 
Regards

**Pull request merged**